spec/whoop_spec.rb

```diff
@@ -88,6 +98,19 @@
         expect(logged_message).not_to include(context)
       end
     end
+
+    context 'parsing PostgreSQL operators' do
+      it 'appropriately formats jsonb column operators' do
+        io = setup_whoop
+
+        ['->>', '->' , '#>>', '#>'].each do |token|
```

Comment:
I wonder if these tokens might be insufficient for testing actual SQL queries. I suggest changing this list to use actual SQL derived from the Postgres docs. For example:

```ruby
[
    %Q['{"a": {"b":"foo"}}'::json->'a'],
    %Q['[1,2,3]'::json->>2],
    %Q['{"a":1,"b":2}'::json->>'b'],
    %Q['{"a": {"b":{"c": "foo"}}}'::json#>'{a,b}'],
    %Q['{"a":[1,2,3],"b":[4,5,6]}'::json#>>'{a,2}'],
    # ... all examples from https://www.postgresql.org/docs/9.5/functions-json.html
]
```